tests/pages/bigtest/package-show.js

```diff
@@ -0,0 +1,23 @@
+import {
+  clickable,
+  isPresent,
+  page,
+  property,
+} from '@bigtest/interaction';
+
+@page class PackageShowModal {
+  confirmDeselection = clickable('[data-test-eholdings-package-deselection-confirmation-modal-yes]');
+  cancelDeselection = clickable('[data-test-eholdings-package-deselection-confirmation-modal-no]');
+}
+
+@page class PackageShowPage {
+  allowKbToAddTitles = property('checked', '[data-test-eholdings-package-details-allow-add-new-titles] input');
+  hasAllowKbToAddTitles = isPresent('[data-test-eholdings-package-details-toggle-allow-add-new-titles] input');
+  hasAllowKbToAddTitlesToggle = isPresent('[package-details-toggle-allow-add-new-titles-switch]');
+  isSelected = property('checked', '[data-test-eholdings-package-details-selected] input');
+  modal = new PackageShowModal('#eholdings-package-confirmation-modal');
```

Comment:
Can we access the modal with a `data-test` attribute instead of relying on an id?

Reply:
We can revisit later.